### ISSUE TITLE
Editable package "v" makes newest pipenv fail

### ISSUE THREAD

**Issue by jameslan** [danielgtaylor/python-betterproto#42]
In Pipfile, there's such line in `dev-packages`:

```
v = {editable = true,version = "*"}
```

pipenv 2018.11.26 doesn't complain but the head of pipenv master branch will fail because an editable package should be a local path or git repo URL.

Also,

```sh
$ pip install -e v
ERROR: v is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with svn+, git+, hg+, or bzr+).
```

**Comment by jameslan** [comment]
Forgot to metion

**Comment by boukeversteegh** [comment]
What is this package? I see a placeholder mentioned on pip:

https://pypi.org/project/v/

Can we simply remove this line?

**Comment by boukeversteegh** [comment]
@nat-n Would you know if this line is needed?

**Comment by nat-n** [comment]
Worth a shot, might have actually been a typo via the cli  😳

**Comment by nat-n** [comment]
Addressed in PR#72
